technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -220,20 +213,11 @@
     <choose>
       <if type="chapter paper-conference entry-encyclopedia entry-dictionary song musical_score" match="any">
         <text term="in" suffix=": "/>
-        <text variable="container-title" font-style="italic" suffix=", "/>
         <group>
           <text macro="container-contributors" suffix=", "/>
-          <choose>
-            <if variable="volume">
-              <group delimiter=" " suffix=", ">
-                <text variable="number-of-volumes"/>
-                <text term="number-of-volumes" form="short"/>
-              </group>
-              <text term="volume" form="short" suffix=" "/>
-              <text variable="volume" suffix=", "/>
-            </if>
-          </choose>
-          <text macro="secondary-container-contributors" prefix=", "/>
+          <text variable="container-title" font-style="italic" suffix=", "/>
+          <text macro="volume" suffix=", "/>
+          <text macro="secondary-container-contributors" suffix=", "/>
         </group>
```

Comment:
@jmiba, assuming you're editing by hand, I recommend keeping an eye out for situations where you can replace prefixes and suffixes by group delimiters, which are generally more robust and preferred. E.g. you can rewrite:

```xml
        <text term="in" suffix=": "/>
        <group>
          <text macro="container-contributors" suffix=", "/>
          <text variable="container-title" font-style="italic" suffix=", "/>
          <text macro="volume" suffix=", "/>
          <text macro="secondary-container-contributors" suffix=", "/>
        </group>
```

as

```xml
        <group delimiter=": " suffix=", ">
          <text term="in"/>
          <group delimiter=", ">
            <text macro="container-contributors"/>
            <text variable="container-title" font-style="italic"/>
            <text macro="volume"/>
            <text macro="secondary-container-contributors"/>
          </group>
        </group>
```